draft-gpew-priv-ppm.md

```diff
@@ -1126,8 +1122,8 @@ the report sequence any report for which the initial state is FAILED.
 
 Let `[aggregator]` denote the helper's API endpoint. The leader extracts the
 sequence of report shares for `[aggregator]` and sends sends a POST request to
-`[aggregator]/helper/aggregate_init` with an Aggregate message of type
-`agg_init_req` and with AggregateInitReq as the payload.
+`[aggregator]/aggregate_init` with an Aggregate message of type `agg_init_req`
```

Comment:
Do we need separate `aggregate` and `aggregate_init` endpoints? `struct Aggregate` contains an `AggregateType` selector that allows the recipient to tell an `AggregateInitReq` from an `AggregateReq` anyway.

Reply:
Right, this may not be strictly necessary.